tests/buffer/buffer_data_tracking.cpp

```diff
@@ -0,0 +1,160 @@
+/* RUN: %{execute}%s | %{filecheck} %s
+   CHECK: Result:
+   CHECK-NEXT: 0 0 0
+   CHECK-NEXT: 3 4 5
+   CHECK-NEXT: 1 1 1
```

Comment:
This test does not pass with LIT from `Makefile` because you forgot to output a `std::endl` after each test.